<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSettings</class>
 <widget class="QWidget" name="FormSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Настройки</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QCheckBox" name="checkBoxVertLine">
          <property name="text">
           <string>вертикальные линии</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QCheckBox" name="checkBoxFog">
          <property name="text">
           <string>туман</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>сетевой обмен</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QCheckBox" name="checkBoxILS">
          <property name="text">
           <string>отрисовка ИЛС</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="checkBoxInfo">
          <property name="text">
           <string>информация</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Масштаб</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="7" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Частота кадров(отрисовка)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBoxFreq">
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>20</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="8" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Множитель времени</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="doubleSpinBoxMulTime">
            <property name="minimum">
             <double>-100.000000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="9" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Интерполяция</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QRadioButton" name="radioButtonLinear">
             <property name="text">
              <string>линейная</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="radioButtonSpline">
             <property name="text">
              <string>сплайн</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="checkBoxLand">
          <property name="text">
           <string>отрисовка ландшафта</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Загрузка</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonLoadData">
          <property name="text">
           <string>данные полета</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButtonLoadLand">
          <property name="text">
           <string>ландшафт</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
